.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,7 @@
 ## PR Author Checklist
 
+- [ ] PR title adheres to [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#specification)
+- [ ] All commits in this PR adhere to [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#specification)
```

Comment:
I think the first item with PR title is good. The second checklist item (line 4) isn't something that can be fixed within the scope of the PR if the code has already been committed. I think in that scenario we need to be covering that by reviewing code and coaching the team onto the right path with the commit messages. The bigger concern is the title which will end up becoming the single, historical commit in the history.

I think we should still be following conventional commits in the other commits that get squashed, but probably not include it in the PR checklist.
```suggestion
```
```suggestion
- [ ] All commits in this PR adhere to [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/#specification)
```